wiring/src/spark_wiring_ble.cpp

```diff
@@ -2177,8 +2168,11 @@ ssize_t BleLocalDevice::getAdvertisingData(BleAdvertisingData* advertisingData)
     if (advertisingData == nullptr) {
         return SYSTEM_ERROR_INVALID_ARGUMENT;
     }
+    hal_ble_adv_params_t advParams = {};
```

Comment:
```c++
advParams.size = sizeof(hal_ble_adv_params_t);
```

Reply:
And initializing the `.version` field would be even better.